providers/printers.rb

```diff
@@ -9,6 +9,55 @@
 # http://www.osor.eu/eupl
 #
 
+def install_printer(prt_name, prt_id, prt_model, prt_uri, prt_policy)
+	print "installing printer #{prt_name}... "
+	printer_drv = `foomatic-ppdfile -P #{prt_model}|grep "'#{prt_id}'"`.scan(/CompatibleDrivers='(\S+)\s.*'/)
```

Comment:
`foomatic-ppdfile` needs both arguments, `-p` and `-d` to successfully get PPD file. I've run tests looking for a optimized way of getting the file but it just works like this. Plus, `-P` argument (capital P) is used for searching purposes and `-p` (minuscule letter) is used for getting the PPD file. You need the driver or drivers name in order to get the file.